packages/core/src/scope.ts

```diff
@@ -365,6 +375,9 @@ export class Scope implements ScopeInterface {
       if (captureContext.requestSession) {
         this._requestSession = captureContext.requestSession;
       }
+      if (captureContext.propagationContext) {
+        this._propagationContext = captureContext.propagationContext;
+      }
```

Comment:
Hmm seems like we have quite some duplicated logic here when distinguishing between `captureContext` being a class instance or a plain object. No action required in this PR but we should probable deduplicate this.